src/youtube-player/BUILD.bazel

```diff
@@ -19,6 +19,7 @@ ng_module(
     ),
     module_name = "@angular/youtube-player",
     deps = [
+        "//src:typings",
```

Comment:
I'm open to renaming it since I'm not a fan of `typings` either. My thinking was that we could put other global typings here if necessary. On `core` this is called `utilities`, but that's not that good of a name either.

Reply:
+1 to `dev_mode_types`, `typings` is a bit broad for me.